services/order.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+from typing import List
+from django.db.models import QuerySet
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from db.models import Order, Ticket, MovieSession
+
+
+def create_order(tickets: List[dict], username: str, date: str = None) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user_id=user.id)
+        if date:
+            order.created_at = datetime.strptime(date, "%Y-%m-%d %H:%M")
+            order.save()
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                order=order,
+                movie_session=MovieSession.objects.get(
+                    id=ticket.get("movie_session")
+                ),
+                row=ticket.get("row"),
+                seat=ticket.get("seat")
+            )
+        return order
+
+
+def get_orders(username: str = None) -> QuerySet:
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]